docs/io/fits/usage/image.rst

```diff
@@ -33,22 +31,37 @@ the numpy array of its data will have the shape of (400, 300).
 
 Here is a summary of reading and updating image data values::
 
-    >>> f = fits.open('image.fits')  # open a FITS file
-    >>> scidata = f[1].data  # assume the first extension is an image
-    >>> print(scidata[1, 4])   # get the pixel value at x=5, y=2
-    >>> scidata[30:40, 10:20]  # get values of the subsection
-    ...                        # from x=11 to 20, y=31 to 40 (inclusive)
-    >>> scidata[1,4] = 999  # update a pixel value
-    >>> scidata[30:40, 10:20] = 0  # update values of a subsection
-    >>> scidata[3] = scidata[2]    # copy the 3rd row to the 4th row
+    >>> from astropy.io import fits
+    >>> fits_image_filename = fits.util.get_testdata_filepath('test0.fits')
+
+    >>> with fits.open(fits_image_filename) as hdul:  # open a FITS file
+    ...     data = hdul[1].data  # assume the first extension is an image
+    >>> print(data[1, 4])   # get the pixel value at x=5, y=2
+    313
+    >>> # get values of the subsection from x=11 to 20, y=31 to 40 (inclusive)
+    >>> data[30:40, 10:20]
+    array([[314, 314, 313, 312, 313, 313, 313, 313, 313, 312],
+           [314, 314, 312, 313, 313, 311, 313, 312, 312, 314],
+           [314, 315, 313, 313, 313, 313, 315, 312, 314, 312],
+           [314, 313, 313, 314, 311, 313, 313, 313, 313, 313],
+           [313, 314, 312, 314, 312, 314, 314, 315, 313, 313],
+           [312, 311, 311, 312, 312, 312, 312, 313, 311, 312],
+           [314, 314, 314, 314, 312, 313, 314, 314, 314, 311],
+           [314, 313, 312, 313, 313, 314, 312, 312, 311, 314],
+           [313, 313, 313, 314, 313, 313, 315, 313, 312, 313],
+           [314, 313, 313, 314, 313, 312, 312, 314, 310, 314]], dtype=int16)
+    >>> data[1,4] = 999  # update a pixel value
+    >>> data[30:40, 10:20] = 0  # update values of a subsection
+    >>> data[3] = data[2]    # copy the 3rd row to the 4th row
 
 Here are some more complicated examples by using the concept of the "mask
-array". The first example is to change all negative pixel values in scidata to
+array". The first example is to change all negative pixel values in data to
```

Comment:
Should `data` be double-backticked here?

Reply:
Yes, good catch.